### ISSUE TITLE
Finetuned weights don't work

### ISSUE THREAD

**Issue by Louma20** [DepthAnything/Depth-Anything-V2#250]
I finetuned metric depth on my own dataset, when I saved the checkpoint and run `metric_depth/run.py`, I got:
```
model.load_state_dict(torch.load("/content/latest.pth", map_location='cpu'))
model = model.to(DEVICE).eval()
Unexpected key(s) in state_dict: "model", "optimizer", "epoch", "previous_best". 

```
